### PR TITLE
Draft & Publish documentation

### PR DESCRIPTION
**Pull request by meganelacheny**
- Documentation of the Draft & Publish feature
- Addition of FA icons

### CODE REVIEW

**Review by lauriejim — CHANGES_REQUESTED** [review]
The Concept menu is ordered by Alphabetic, can you make sure this order is respected.

File: docs/v3.x/concepts/draft-and-publish.md [hunk 1]
```diff
@@ -14,28 +14,36 @@ To deactivate the draft and publish mode for a content type:
...
 
 ### Publishing a draft
 
 To publish a draft, click on the **Publish** button.
 
+:::warning CAUTION
+Before publishing a draft, make sure it doesn't have relations with other non-published content, otherwise some of the content may not be available through the API.
+:::
+
 When a content is not in a draft anymore, but a published content, it is indicated on the right of the interface, below the Information box.
```
Comment (derrickmehaffy) [inline docs/v3.x/concepts/draft-and-publish.md:44]: > When content is not in a draft state anymore, but a published state, it is indicated on the right of the interface, below the information box.

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Review by lauriejim — APPROVED** [review]
LGMT

**Review by derrickmehaffy — APPROVED** [review]
LGTM

File: docs/v3.x/concepts/draft-and-publish.md [hunk 2]
```diff
@@ -35,18 +35,18 @@ Drafts can be modified and saved at will, until they are ready to be published.
 
 ### Publishing a draft
 
-To publish a draft, click on the **Publish** button.
+To publish a draft, click on the **Publish** button in the top right corner of the content edition interface.
```
Comment (derrickmehaffy) [inline docs/v3.x/concepts/draft-and-publish.md:38]: > content edition

Do you mean "content editor" ?

File: docs/v3.x/concepts/draft-and-publish.md [hunk 3]
```diff
@@ -35,18 +35,18 @@ Drafts can be modified and saved at will, until they are ready to be published.
...
 
 :::warning CAUTION
 Before publishing a draft, make sure it doesn't have relations with other non-published content, otherwise some of the content may not be available through the API.
 :::
 
-When a content is not in a draft anymore, but a published content, it is indicated on the right of the interface, below the Information box.
+When a content is not a draft anymore, but a published content, it is indicated on the right of the interface, below the Information box.
```
Comment (derrickmehaffy) [inline docs/v3.x/concepts/draft-and-publish.md:44]: > but a published content

This needs to be changed but I'm not sure of the best way to word it :thinking: maybe "but has been published"

File: docs/v3.x/concepts/draft-and-publish.md [hunk 4]
```diff
@@ -35,18 +35,18 @@ Drafts can be modified and saved at will, until they are ready to be published.
...
 ![Editing published version](../assets/concepts/draft-publish/editing_published_version.png)
 
 ### Unpublishing content
 
-Published contents can be unpublished, switching back to drafts again.
+Published contents can be unpublished, switching back to being drafts again.
 
-To unpublish content, click on the **Unpublish** button.
\ No newline at end of file
+To unpublish content, click on the **Unpublish** button in the top right corner of the content edition interface.
```
Comment (derrickmehaffy) [inline docs/v3.x/concepts/draft-and-publish.md:52]: same here, should that be "content editor" ?

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Comment by soupette** [comment]
I am not sure we need all 980 fontawesome icons in the documentation. The most used ones are ` faPlus, faTrashAlt, faPencil` ...

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Pull request merged**